evergreen/convert-client-cert-to-pkcs12.sh

```diff
@@ -25,6 +25,21 @@ openssl pkcs12 -export -in "${CLIENT_PEM}" \
   -name "Drivers Client Certificate" \
   -password "pass:${MONGO_X509_CLIENT_CERTIFICATE_PASSWORD}"
 
+if [[ "$OS" =~ MAC|Mac|mac ]]; then
```

Comment:
`realpath` is missed in mac bash, so workaround from here: https://github.com/whatwg/html-build/issues/90#issuecomment-235960102 (also discussed it with Ryan Egesdahl from #server-builds)